packages/flutter/lib/src/rendering/object.dart

```diff
@@ -4361,13 +4366,14 @@ class _IncompleteSemanticsFragment extends _InterestingSemanticsFragment {
 class _SwitchableSemanticsFragment extends _InterestingSemanticsFragment {
   _SwitchableSemanticsFragment({
     required bool mergeIntoParent,
+    required bool blockUserInteractions,
```

Comment:
nit: it is a little odd that this constructor takes blockUserInteractions as a param and then just turns around to modify another param. Should we just edit the config before it is passed here>